scripts/local/docker_memory_check.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+import argparse
+import json
+import os.path
+import sys
+from typing import Sequence
+
+
+def should_use_color(setting: str) -> bool:
+    # normally I would use `sys.stdout.isatty()` however direnv always pipes this
+    return setting == "always" or (setting == "auto" and not os.environ.get("CI"))
+
+
+def color(s: str, color: str, *, use_color: bool) -> str:
+    if use_color:
+        return f"{color}{s}\033[m"
+    else:
+        return s
+
+
+def main(argv: Sequence[str] | None = None) -> int:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--settings-file",
+        default=os.path.expanduser("~/Library/Group Containers/group.com.docker/settings.json"),
+        help=argparse.SUPPRESS,
+    )
+    parser.add_argument(
+        "--memory-minimum",
+        default=8092,
```

Comment:
I know this looks like a typo -- this is the value docker.app sets by default currently (I guess they don't know their bitwise numbers!) and so I picked that to avoid notifying *everyone*